base/strings/string_util_starboard.h

```diff
@@ -54,11 +54,13 @@ inline int strncmp16(const char16* s1, const char16* s2, size_t count) {
 #endif
 }
 
+#if SB_API_VERSION <= 15
 inline int vswprintf(wchar_t* buffer, size_t size,
                      const wchar_t* format, va_list arguments) {
   DCHECK(base::IsWprintfFormatPortable(format));
   return SbStringFormatWide(buffer, size, format, arguments);
```

Comment:
We should remove the use of the API everywhere and switch to the POSIX equivalent.
In musl we should take care of backwards compatibility for older SB versions for Evergreen/Modular platforms.
And for win32 based platforms we'll have to add the POSIX API if it is missing.